client/components/data/query-reader-list-items/index.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestReaderListItems } from 'state/reader/lists/actions';
+
+class QueryReaderListItems extends Component {
```

Comment:
there's a more modern form for this. see https://github.com/Automattic/wp-calypso/blob/master/client/components/data/query-embed/index.jsx

Reply:
Thanks! The new style is much nicer.